smart_card_connector_app/build/human_readable_supported_readers.txt

```diff
@@ -13,6 +13,7 @@ ACS ACR1251 Dual Reader
 ACS ACR1252 Dual Reader
 ACS ACR1252 Reader
 ACS ACR1255U-J1
+ACS ACR1252IMP Reader
```

Comment:
nit: Hmm, I think this should come before the previous item, since the lines are intended to be sorted in lexicographical order.

(The Python script that generates this file should do this automatically, but you probably didn't use it? That's fine, just trying to understand whether we have a bug in this script or not.)